src/ui/TrackViewer.tsx

```diff
@@ -153,6 +156,7 @@ export class TrackViewer extends Object2D {
         // hide/show add panel button
         let clampToTracks = state.clampToTracks == null ? false : state.clampToTracks;
         this.allowNewPanels = state.allowNewPanels == null ? false : state.allowNewPanels;
+        this.reorderTracks = state.reorderTracks == null ? false : state.reorderTracks;
```

Comment:
This is fine for consistency. But I would of had done:

this.reorderTracks = !!state.reorderTracks;

Reply:
This is better.